app/controllers/admin/case_studies_controller.rb

```diff
@@ -0,0 +1,66 @@
+class Admin::CaseStudiesController < ApplicationController
+	layout 'admin_panel_sidenav'
+  before_action :authenticate_user!
+	before_action :require_admin, only: [:index, :update, :edit, :show, :destroy]
+	before_action :set_case_study, only: [:edit, :update, :show, :destroy]
+
+	def index
+    @case_studies = CaseStudy.all.includes(:created_by).order(created_at: :desc).filter_by(params[:page], filter_params.to_h, sort_params.to_h)
+    respond_to do |format|
+      if request.xhr?
+        format.html {render partial: 'admin/case_studies/table', locals: { case_studies: @case_studies } }
+      else
+        format.html
+      end
+    end
+	end
+
+	def show
+	end
+
+	def update
+		if @case_study.update(secure_params)
+			redirect_to admin_case_study_path(@case_study), flash_success_info: 'Case Study details was successfully updated.'
+		else
+			redirect_back fallback_location: root_path, flash_info: 'Case Study details was not successfully updated.'
+		end
+	end
+
+	def destroy
+		@case_study.destroy
+		redirect_to admin_case_studies_path, flash_success_info: 'Case Study was successfully deleted.'
+	end
+
+	def new
+		@case_study = CaseStudy.new
+	end
+
+	def create
+    @case_study = CaseStudy.new(secure_params.merge(created_by_id: current_user.id))
```

Comment:
```suggestion
    @case_study = CaseStudy.new(secure_params.merge(created_by: current_user))
```

can you check if this works?